leptos_dom/src/html.rs

```diff
@@ -276,6 +276,12 @@ impl ElementDescriptor for Custom {
 cfg_if! {
   if #[cfg(all(target_arch = "wasm32", feature = "web"))] {
     /// Represents an HTML element.
+    /// Beginner's tip: 
+    /// HtmlElement<El> implements Deref where El implements Deref.
+    /// When El has a corresponding web_sys::HtmlElement -> El will implement Deref for it.
```

Comment:
Currently, all of these are listed as plain text. If you make them links like \[\`web_sys::HtmlElement\`\] then they will link to the appropriate docs. This could be done for `HtmlElement` and `Deref` at the least. Other types can be wrapped in backticks \`\` so they appear as code not plain text in the docs.